store/tikv/gcworker/gc_worker.go

```diff
@@ -106,6 +106,10 @@ const (
 	gcDefaultLifeTime        = time.Minute * 10
 	gcSafePointKey           = "tikv_gc_safe_point"
 	gcSafePointCacheInterval = tikv.GcSafePointCacheInterval
+	gcConcurrencyKey         = "tikv_gc_concurrency"
+	gcDefaultConcurrency     = 32
```

Comment:
As i have tested, the concurrency should less than 50% of the total count of TiKV nodes.

Reply:
I test a cluster with 5 tikv node and 2000 regions, when i set GC concurrency as 2, only 5 regions will be skipped during a round of GC.

Reply:
It is somehow different from what I think it will do, but I'm glad it works well.